### ISSUE TITLE
Bug when ignoring imports that are in more than one place in a module

### ISSUE THREAD

**Issue by sobolevn** [seddonym/import-linter#108]
Hi!

I am using `import-linter` for quite a long time now. It always showed that my project has no problems.
But, after `1.2.5` was released, it shows this:

```
 poetry run lint-imports
The edge wemake_python_styleguide.formatter-pygments not in graph.
```

Related https://github.com/wemake-services/wemake-python-styleguide/pull/2188

Is it a valid issue? Or a regression?

**Comment by seddonym** [comment]
Thanks for raising @sobolevn! Sounds like a bug. The way `ignore_imports` are handled has changed slightly in the latest release. I'll try to get to the bottom of this over the next few days.

**Comment by seddonym** [comment]
I see the bug now. The import to `pygments` is happening [multiple times](https://github.com/wemake-services/wemake-python-styleguide/blob/dependabot/pip/import-linter-1.2.5/wemake_python_styleguide/formatter.py#L34-L36). Import Linter tries to remove it from the graph each time, so the second time it will not be present.

I'll aim to fix this in the next few days.

**Comment by seddonym** [comment]
This is now fixed in 1.2.6 - thanks for raising it.

**Comment by sobolevn** [comment]
Thanks for the fix! 👍
